CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- There should always be "Unreleased" section at the beginning. -->
 
 ## Unreleased
+- [**BC**] Require php 8.1
```

Comment:
(The BC is not requiring PHP 8.1, its using the language features & change method signatures :) .)